framework/lib/utils/debounce/create-debounce-function-instance.ts

```diff
@@ -0,0 +1,18 @@
+/** A debounce function to be used together with Async select in SearchBar.
```

Comment:
You wrote this? ;)

Reply:
Adjusted a stackoverflow suggestion.
Chatgpt actually only suggested debounce from lodash